packages/tinacms/src/components/TinaProvider.test.tsx

```diff
@@ -1,12 +1,17 @@
 import { render } from '@testing-library/react'
 import React from 'react'
-import { TinaProvider, CMS } from '../../build'
-import { TinaCMS } from '..'
+import { TinaProvider, INVALID_CMS_ERROR } from './TinaProvider'
+import { TinaCMS } from '../tina-cms'
+import { CMS } from '@tinacms/core'
 
 describe('TinaProvider', () => {
   describe('when passed an instance of CMS', () => {
     it('throws no error', () => {
-      render(<TinaProvider cms={new CMS()} />)
+      const t = () => {
+        render(<TinaProvider cms={new CMS() as any} />)
+      }
+
+      expect(t).toThrowError(INVALID_CMS_ERROR)
```

Comment:
You're expecting to throw an error here, but the `it` block above still states it throws no error.